rskj-core/src/main/java/org/ethereum/vm/VM.java

```diff
@@ -1486,22 +1413,6 @@ protected void doCALL(){
 
         callToAddress(codeAddress, msg);
 
-        program.disposeWord(inDataOffs);
-        program.disposeWord(inDataSize);
-        program.disposeWord(outDataOffs);
-        program.disposeWord(outDataSize);
-        program.disposeWord(codeAddress);
-        program.disposeWord(gas);
-        if (config.isRskip103()) {
```

Comment:
I some behavior here must be kept for consensus. Have you tried syncing mainnet and testnet with this code?

Reply:
The removed disposeWord doesn't alter consensus.